docs/Changelog.md

```diff
@@ -42,6 +42,7 @@
 * 1.12.5 (1 Dec 2023)
 ------------------------
 * [#5110](https://github.com/TouK/nussknacker/pull/5110) Fix: The compare option doesn't display fragment input properties between the two versions
+* [#5241](https://github.com/TouK/nussknacker/pull/5241) FragmentParameter uses `ValidationExpressionParameterValidator` through new field: `valueCompileTimeValidation`
```

Comment:
It won't be added to 1.12, we almost have 1.13! :) Please bundle it with `Expand FragmentParameter`. TBH both of them don't look like comments for a human being. Can we do sth with this?

Reply:
i've changed and batched  it with  changelog of https://github.com/TouK/nussknacker/pull/4978

(the 1.12.5 was barely visible because of the *, fixed it)